### DISCUSSION TITLE
FN5 Feedback

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#129] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/FN5.html 

Author: @AbbyANoble 
Reviewer: @kathypinzon

**Comment by kathypinzon** [comment]
Activity 2.5.2 I would keep the directions the same as 2.5.1 (most simplified form).

**Comment by kathypinzon** [comment]
Activity 2.5.4 I would suggest reminding them of how to evaluate a function. Either with a remark before or maybe make part a be something like how do you evaluate f(3)?

Another plausible wrong answer could be "multiply 3x-5 by x^3-2"

**Comment by kathypinzon** [comment]
I am assuming there is more coming to this activity....... ;)

**Comment by AbbyANoble** [comment]
Finished the draft in #135!

**Comment by kathypinzon** [comment]
2.5.4 part (d) is missing a question mark at the end.

**Comment by kathypinzon** [comment]
Remark 2.5.6 has a typo near the end "and then outer" should be "and the outer"

**Reply by AbbyANoble** [reply]
Good catch. I went with "and then the outer".

**Comment by kathypinzon** [comment]
Activity 2.5.7 (b) I don't see the correct answer.  Shouldn't it be \( \left( \frac{1}{3x-5) \right)^{3} -2\)

**Reply by AbbyANoble** [reply]
Oops. And in part (a). I meant for g(f(x)) to be answer choice for both a and b.
